site2/docs/deploy-monitoring.md

```diff
@@ -69,6 +69,26 @@ brk_ml_cursor_persistZookeeperErrors(namespace="", ledger_name="", cursor_name:"
 
 Those metrics are added in the Prometheus interface, you can monitor and check the metrics stats in the Grafana.
 
+### Function & connector stats
+
+You can collect functions worker stats from `functions-worker` and export the metrics in JSON formats, which contain functions worker JVM metrics.
+
+```
+pulsar-admin functions-worker monitoring-metrics
+```
+
+You can collect functions and connectors metrics from `functions-worker` and export the metrics in JSON formats.
+
+```
+pulsar-admin functions-worker function-stats
+```
+
+The aggregated functions and connectors metrics can be exposed in Prometheus formats as below. You can get `FUNCTIONS_WORKER_ADDRESS` and `WORKER_PORT` from the `functions_worker.yml` file.
```

Comment:
@freeznet I add `FUNCTIONS_WORKER_ADDRESS` here, is it correct?

Reply:
`FUNCTIONS_WORKER_ADDRESS` is not in `functions_worker.yml`, it is related how user config their functions worker, related to [http://pulsar.apache.org/docs/en/next/functions-worker/](http://pulsar.apache.org/docs/en/next/functions-worker/).

Reply:
@freeznet thanks for the explanations. Should we need to tell readers how to get `FUNCTIONS_WORKER_ADDRESS` here?

Reply:
@freeznet any feedback?

Reply:
I suggest to add a link to functions-worker doc, should enough to let reader understand.

Reply:
@freeznet OK, I've added.